ngsderive/commands/endedness.py

```diff
@@ -197,16 +200,23 @@ def main(
                 raise RuntimeError(
                     "This shouldn't be possible. Please contact the developers."
                 )
-        assert (
-            ordering_flags["overall"]["firsts"]
-            + ordering_flags["overall"]["lasts"]
-            + ordering_flags["overall"]["neither"]
-            + ordering_flags["overall"]["both"]
-        ) > 0
+
+        rgs_in_header_not_in_seq = validate_read_group_info(
+            {
+                rg
+                for rg in ordering_flags
+                if rg not in {"overall", "unknown_read_group"}
```

Comment:
These should be extracted out into constants and the entire set should be extracted out to some global set (so that the values are not hardcoded so deep down in the code).

Reply:
Any other places where the key is referenced should also be updated to use the constant(s).

Reply:
That's a good idea. Will make that update.